src/CFamily.UnitTests/CMake/ActiveConfigProviderTests.cs

```diff
@@ -0,0 +1,120 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.IO;
+using System.IO.Abstractions;
+using FluentAssertions;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using SonarLint.VisualStudio.CFamily.CMake;
+using SonarLint.VisualStudio.Integration;
+using SonarLint.VisualStudio.Integration.UnitTests;
+
+namespace SonarLint.VisualStudio.CFamily.UnitTests.CMake
+{
+    [TestClass]
+    public class ActiveConfigProviderTests
+    {
+        private const string ExpectedDefaultConfig = "x64-Debug";
```

Comment:
Maybe make the const in the class Internal and re-use it?

Reply:
I prefer to have unit tests checking explicitly for the expected value. If the test is checking a constant on the product code then there is nothing checking that the constant has the correct value - it could contain anything, and the test would still pass.